src/Table/index.js

```diff
@@ -30,7 +42,7 @@ Table.Body.displayName = "TableBody";
 
 Table.HeaderCell = React.forwardRef(({ children, ...props }, ref) => (
   <th
-    tw="px-6 py-2 bg-white text-left text-xs leading-5 font-medium text-accent-five uppercase tracking-wider"
+    tw="px-6 first:pl-0 last:pr-0 py-2 bg-white text-left text-xs leading-5 font-medium text-accent-five uppercase tracking-wider"
```

Comment:
We don't need this anymore, right?

Reply:
Actually we did to set the default style,  but thanks for pointing out. I moved this to the same block where we add the optional padding.